app/(app)/offence/photos/detail.tsx

```diff
@@ -74,20 +78,28 @@ const AppRoute = () => {
     <ScreenView
       hasBackButton
       title={t('offence.picture.detail.title')}
-      options={{
-        headerRight: () => (
-          <IconButton
-            accessibilityLabel={t('offence.picture.detail.retake')}
-            name="cached"
-            onPress={retakePicture}
-          />
-        ),
-      }}
+      options={
+        index
```

Comment:
For me, the name `index` is kind of confusing, because it usually refers to array index.